### PR TITLE
Enable generic method parsing by default

### PR DESCRIPTION
**Pull request by kevmoo**
Running tests locally now...

### CODE REVIEW

File: pkg/analyzer/pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: analyzer
-version: 0.29.6
+version: 0.29.6+1
```
Comment (bwilkerson) [inline pkg/analyzer/pubspec.yaml]: I don't understand the "+1". I was planning on making it "0.29.7". Can you explain the difference?

**Review by bwilkerson — APPROVED** [review]
This looks fine, as far as it goes, but Phil also needs to make a small change before we actually publish (unless we can't wait and have to publish twice).

File: pkg/analyzer/pubspec.yaml (hunk 1 above)
Comment (kevmoo) [inline pkg/analyzer/pubspec.yaml]: I guess it could/should be `.7`

There is no API change, but there is a behavior change.

**Comment by kevmoo** [comment]
there are 7 failed tests (11693 passed)

All in the form

```
FAIL: analysis engine | generated tests | DirectoryBasedDartSdkTest | test_[fromFile_XYZ | getXYZ]
  Expected: not null
    Actual: <null>
```

File: pkg/analyzer/pubspec.yaml (hunk 1 above)
Comment (kevmoo) [inline pkg/analyzer/pubspec.yaml]: Sent an update

**Comment by bwilkerson** [comment]
Did the tests pass before your change?

**Comment by kevmoo** [comment]
> Did the tests pass before your change?

You ask good questions!

**Comment by kevmoo** [comment]
@bwilkerson Same failures before/after.

**Comment by pq** [comment]
> This looks fine, as far as it goes, but Phil also needs to make a small change before we actually publish (unless we can't wait and have to publish twice).

Yes.  Please!

Could you change the constraint on `isolate`?

    isolate: '>=0.2.2 <2.0.0'

Thanks!

**Review by bwilkerson — APPROVED** [review]
In that case, I'm going to guess that some previous change caused the break, and I'd say you should go ahead and merge this (possibly after addressing Phil's ask).

**Comment by kevmoo** [comment]
@pq @bwilkerson Reran tests with the latest isolate pkg. No new failures. I think we're good. PTAL

File: pkg/analyzer/pubspec.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ dependencies:
   crypto: '>=1.1.1 <3.0.0'
   glob: ^1.0.3
   html: '>=0.12.0 <1.14.0'
-  isolate: ^0.2.2
+  isolate: '>=0.2.2 <2.0.0'
```
Comment (pq) [inline pkg/analyzer/pubspec.yaml]: \o/

Thanks!

**Review by pq — APPROVED** [review]

**Comment by bwilkerson** [comment]
lgtm

**Pull request merged**